### ISSUE TITLE
Peer feedback on dataproject from wkg579

### ISSUE THREAD

**Issue by ghost** [NumEconCopenhagen/projects-2019-raj#2]
**1. The best part of the project was: **

I believe the heatmaps were the best part, they are a great way of visualizing data and showing patterns. I also enjoy that you have quite a lot of text throughout the assignment, such that everything is explained.

**2. The hardest part of the project to understand was:**

I have had Social Data Science, and thus I have been taught machine learning in Python. However, that is still the most cumbersome part of the project.  

**3. An idea for an improvement/clarification could be:**

You have several plots where the variable displayed on the 1. axis only takes on whole positive integers, yet your x-ticks are set to increments of less than one. I would recommend changing that.

Regarding your machine learning, I am not sure that splitting a data set into a training and a testing dataset in and of it itself is machine learning. As far as I can see, you do regular OLS on the training dataset, and the your predict the testing dataset. This is an out of sample prediction and surely is interesting, but I am not sure it counts as machine learning. You could e.g. do regularized OLS and trained to achieve the best hyperparameter and precision in the training dataset.

**4. An idea for an extension could be:**

Instead of doing your machine learning part which does not really seem to be part of the course (even though it is interesting), I would try to create interactive plots (sorry if it seems a bit harsh, but especially parts 4.2 and 4.3 seem kind of out of place. 4.1 as a means of visualizing data is nice, though). Both your heatmaps and histograms would be very nice as interactive plots; in the histograms one could choose their desired variable, in the heatmaps one could choose their desired location.
